### DISCUSSION TITLE
Help -> CP editor just crashes in almost 5 secs after launching

### DISCUSSION THREAD

**Discussion by xHydr1dex** [cpeditor/cpeditor#1360] [Q&A] [answered]
Recently while opening the editor, it kept on closing by itself 5 seconds after launch and getting the cursor stuck on the loading symbol. I have the 7.0.1 ver and until today it was fine and performed well. Should I maybe try to uninstall and reinstall it again?? But won't that take away all the settings including the snippets, cp helper settings and more? Please help.

**Comment by ouuan** [comment]
Are there any open tabs (files)? Try closing all of them.

**Reply by xHydr1dex** [reply]
Please help me on how to close the files from the application, as the app just keeps on crashing without letting me do anything. I am unable to find a way to close the files within the app itself.

**Reply by ouuan** [reply, marked as answer]
Try running `cpeditor --no-restore-session` in the terminal (cmd).

**Reply by ouuan** [reply]
Or you can delete (after backup) the session file. It should be at `C:\Users\<user>\AppData\Local\cpeditor\session.json`.

**Reply by xHydr1dex** [reply]
thanks a lot Yufan!
